### PR TITLE
Replace jshint and jscs by eslint #81

### PR DESCRIPTION
**Pull request by stephane-tessier**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "env": {
+    "node": true,
+    "browser": true,
+    "es6": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {
+      "jsx": true,
```
Comment (ryuran) [inline .eslintrc]: Are you sure for jsx ?

File: .gsk/kss/extend/handlebars/gskHbs.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-/* jshint node:true */
```
Comment (ryuran) [inline .gsk/kss/extend/handlebars/gskHbs.js]: There is no way to make a difference between node js and browser js with eslint ?

Comment (stephane-tessier) [inline .gsk/kss/extend/handlebars/gskHbs.js]: There might be, but i'm thinking that it should rather be at folder level rather than file level, no?

Comment (ryuran) [inline .gsk/kss/extend/handlebars/gskHbs.js]: Maybe. If folder structure allow it.

File: .gsk/conf/readme.md [hunk 3]
```diff
@@ -12,8 +12,7 @@ configurations par défaut, n'hésitez pas à proposer une merge request au
 starter kit pour que l'on en discute.
 
 * `.jsbeautifyrc`  Configuration pour le [formatage automatique du code HTML](https://github.com/beautify-web/js-beautify)
-* `.jscsrc`        Configuration pour la [vérification de syntaxe JS](http://jscs.info/)
-* `.jshintrc`      Configuration pour le [linting des fichiers JS](http://jshint.com/)
+* `.eslint`        Configuration pour [le linting et la vérification de syntaxe des fichiers JS](http://eslint.org)
```
Comment (ryuran) [inline .gsk/conf/readme.md]: rename it in doc `.eslintrc.json`

**Pull request merged**